src/pages/blog/starlingx-deploying-your-app-3.md

```diff
@@ -0,0 +1,250 @@
+
+Now, let's check what is inside the generated folder:
+
+```
+output
+└── poc-starlingx-app
+    ├── charts
+    │   └── poc-starlingx-1.5.2.tgz
+    ├── checksum.sha256
+    ├── fluxcd-manifests
+    │   ├── base
+    │   │   ├── helmrepository.yaml
+    │   │   ├── kustomization.yaml
+    │   │   └── namespace.yaml
+    │   ├── kustomization.yaml
+    │   └── poc-starlingx
+    │       ├── helmrelease.yaml
+    │       ├── kustomization.yaml
+    │       ├── poc-starlingx-static-overrides.yaml
+    │       └── poc-starlingx-system-overrides.yaml
+    ├── metadata.yaml
+    ├── plugins
+    │   ├── __init__.py
+    │   ├── k8sapp_poc_starlingx_app
+    │   │   ├── common
+    │   │   │   ├── constants.py
+    │   │   │   └── __init__.py
+    │   │   ├── helm
+    │   │   │   ├── __init__.py
+    │   │   │   └── poc_starlingx.py
+    │   │   ├── __init__.py
+    │   │   ├── kustomize
+    │   │   │   ├── __init__.py
+    │   │   │   └── kustomie_poc_starlingx_app.py
+    │   │   └── lifecycle
+    │   │       ├── __init__.py
+    │   │       └── lifecycle_poc_starlingx_app.py
+    │   ├── k8sapp_poc_starlingx_app-0.0.0-py2.py3-none-any.whl
+    │   ├── setup.cfg
+    │   └── setup.py
+    └── poc-starlingx-app-1.0.0.tgz
+``` 
+
+As you can see, the helm chart package was packed under the charts subdirectory. The next step would be to configure the plugins, since this is a complicated process and is application-dependent, we recommend you to read the README.MD of the app-gen-tool and reaching for help with the community in the StarlingX mail lists or in the matrix chat.
+
+Please, pay attention to correctly configure the files in `./output/poc-starlingx-app/plugins/k8sapp_poc_starlingx_app`:
+* **helm/poc_starlingx.py**: this file is responsible for integrating the application with the StarlingX system.
+* **kustomize/kustomize_poc_starlingx_app.py**: this file is responsible for updating top-level kustomization resource list.
+* **lifecycle/lifecycle_poc_starlingx_app.py**: this file configures how StarlingX will perform the lifecycle actions for the application opperator.
+
+After the plugin files are configured, I'll run the app-gen-tool command again, but now with the --package-only parameter.
+
+```sh
+(venv)~$: stx-app-generator -i app_manifest.yaml -o ./output --package-only
+
+...
+Plugin wheels generated!
+Checksum generated!
+FluxCD App tarball generated at /home/tomas-barros/blog-post-3/output/poc-starlingx-app/poc-starlingx-app-1.0.0.tgz
+```
+
+Alright! We can see the `poc-starlingx-app-1.0.0.tgz`. Now we only have to send this .tgz to a StarlingX cluster to install it.
+
+## Installing the App
+
+After copying your app package to your StarlingX cluster(in my case a virtual AIO-SX, we can now proceed to install it. But before installing it, please guarantee that your platform-integ-apps application is installed.
```

Comment:
Here, I think this information about your system should be up there where I commented